README.md

```diff
@@ -8,6 +8,43 @@
 
 RStudio addins for [orderly](https://github.com/vimc/orderly)
 
+## Installation
+
+```
+# install.packages("remotes") # if needed
```

Comment:
Can we encourage installing from the vimc drat instead please? As with the orderly instructions? You can leave this in as an alternative though